### PR TITLE
[GSProcessing] Bump version to 0.2.1, add support for arm64 images for EMR Serverless

### PR DESCRIPTION
**Pull request by thvasilo**
*Issue #, if available:*

*Description of changes:*

* Bumps library version to 0.2.1.
* Add arm64 image support for EMR Serverless.
* ~~Add SageMaker Dockerfile with Spark 3.4 for 0.2.1.~~

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/source/gs-processing/usage/distributed-processing-setup.rst [hunk 1]
```diff
@@ -104,13 +104,65 @@ the following to build the SageMaker image:
...
 ``${VERSION}`` will take be the latest available GSProcessing version (e.g. ``0.2.1``).
 
 The script also supports other arguments to customize the image name,
 tag and other aspects of the build. See ``bash docker/build_gsprocessing_image.sh --help``
 for more information.
 
+Support for arm64 architecture
+------------------------------
+
+You might have noticed that we include the image's architecture, ``x86_64`` in the image name.
```
Comment (classicsong) [inline docs/source/gs-processing/usage/distributed-processing-setup.rst:117]: I don't think we need this sentence. We just tell them we support both x86_64 and arm64 in pre-processing with EMR serverless.

File: docs/source/gs-processing/usage/distributed-processing-setup.rst [hunk 2]
```diff
@@ -104,13 +104,65 @@ the following to build the SageMaker image:
...
+For EMR Serverless images, it is possible to build images that support ``arm64`` instances,
+which can lead to improved runtime and cost compared to ``x86_64``. To build ``arm64`` images
```
Comment (classicsong) [inline docs/source/gs-processing/usage/distributed-processing-setup.rst:119]: Can user build arm64 images on arm64 servers?
Qemu emulation may be too complex.

File: graphstorm-processing/docker/0.2.1/sagemaker/Dockerfile.cpu [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# syntax=docker/dockerfile:experimental
```
Comment (classicsong) [inline graphstorm-processing/docker/0.2.1/sagemaker/Dockerfile.cpu:1]: Can you add sagemaker docker support In another PR?

Comment (thvasilo) [reply graphstorm-processing/docker/0.2.1/sagemaker/Dockerfile.cpu:1]: You mean move this entire file to another PR?

File: docs/source/gs-processing/usage/distributed-processing-setup.rst (hunk 2 above)
Comment (thvasilo) [reply docs/source/gs-processing/usage/distributed-processing-setup.rst:119]: They canI can add that as an alternative.

**Review by classicsong — APPROVED** [review]

**Pull request merged**